### PR TITLE
 try statement to track possible instance errors

### PR DESCRIPTION
**Pull request by viniciusdc**

### CODE REVIEW

**Comment by viniciusdc** [comment]
Just an idea

**Comment by beckermr** [comment]
cc @CJ-Wright

File: conda_forge_tick/make_graph.py [hunk 1]
```diff
@@ -203,7 +203,13 @@ def update_nodes_with_bot_rerun(gx):
...
         with node["payload"] as payload:
             for migration in payload.get("PRed", []):
-                pr_json = migration.get("PR", {})
+                try:
+                    pr_json = migration.get("PR", {})
+                    # maybe add a pass check info here ? (if using DEBUG)
+                except Exception as e:
+                    logger.error(
+                        f"BOT-RERUN : could not proceed check with {node}, {e}",
+                    )
```
Comment (CJ-Wright) [inline conda_forge_tick/make_graph.py:212]: I put in `pr_json = None` here just so we always have something asigned to that var.

Comment (viniciusdc) [reply conda_forge_tick/make_graph.py:212]: yes, thanks for that. If not, probably the bot would crash again

**Comment by CJ-Wright** [comment]
I'm currently thinking this is a stopgap measure, we'll need to restore the bad json to working order at some point.

**Comment by beckermr** [comment]
The bot is still off. We need to make sure we reraise this exception

**Comment by CJ-Wright** [comment]
What do you mean by off?

**Comment by CJ-Wright** [comment]
I though the point was to not re-raise?

**Comment by viniciusdc** [comment]
> What do you mean by off?

well it should've come by 2~4 hours ago...

**Comment by viniciusdc** [comment]
> I though the point was to not re-raise?

I thought we would use this as a way to check what exactly is causing the issue

**Comment by CJ-Wright** [comment]
Ahh ok, then please remove my code and add a `raise e`

**Comment by viniciusdc** [comment]
> Ahh ok, then please remove my code and add a `raise e`

isn't this going to stop the process ?

**Comment by CJ-Wright** [comment]
isn't that the point?

**Comment by viniciusdc** [comment]
> isn't that the point?

Ah ok, I got your point now, sorry. Yeah it's indeed

**Comment by viniciusdc** [comment]
just to confirm once merged a PR it's still possible to make changes by committing then ? or it's necessary to open a new one ?

**Comment by CJ-Wright** [comment]
you need to make a new PR

**Comment by viniciusdc** [comment]
pin #1138

**Pull request merged**